focus/operations/src/status.rs

```diff
@@ -8,20 +8,24 @@ use focus_util::app::{App, ExitCode};
 use std::cmp::Ordering;
 use std::{path::Path, sync::Arc, time::Duration};
 
-fn relative_time(current_commit_time: git2::Time, prospective_commit_time: git2::Time) -> String {
+fn relative_time(
+    current_commit_time: git2::Time,
+    prospective_commit_time: git2::Time,
+    primary_branch: String,
+) -> String {
     let difference = prospective_commit_time.seconds() - current_commit_time.seconds();
     let difference_duration = Duration::from_secs(difference.unsigned_abs());
 
-    match prospective_commit_time.cmp(&current_commit_time) {
+    match current_commit_time.cmp(&prospective_commit_time) {
         Ordering::Greater => format!(
-            "{} newer than",
-            humantime::format_duration(difference_duration)
+            "The current commit is {} newer than the upstream {} commit. This can happen when background prefetching has not run yet.",
```

Comment:
current commit is the working_tree head commit. The existing logic is comparing the working _tree head commit to the prefetch head commit. Do you mean compare against "refs/remotes/origin/master" head instead of the prefetch head?

Reply:
Maybe removing this messaging is the better option? Users is able to get this info by running git status.

Reply:
Fair enough, let's get rid of it.